src/lib/utils/clearRefinements.js

```diff
@@ -21,6 +21,7 @@ function clearRefinements({ helper, attributesToClear = [] }) {
   if (attributesToClear.indexOf('query') !== -1) {
     finalState = finalState.setQuery('');
   }
+  finalState = finalState.setPage(0);
```

Comment:
We could have inlined the call into the first assignment.

Reply:
1fc7375